iosHackthonApp/iosHackthonApp/Landing Screen/LandingView.swift

```diff
@@ -0,0 +1,78 @@
+import SwiftUI
+import shared
+
+struct LandingView: View {
+
+    private var constants = Constants()
+    private let sdk = MealsSDK()
+
+    var body: some View {
+        NavigationView {
+            GeometryReader { geometry in
+                VStack {
+                    HStack {
+                        Spacer()
+                        Text(Strings.LandingScreen.subheading)
+                            .font(.subheadline)
+                            .multilineTextAlignment(.center)
+                        Spacer()
+                    }
+                    .padding()
+
+                    NavigationLink(destination: AddMealView()) {
+                        ZStack {
+                            RoundedRectangle(cornerRadius: constants.cornerRadius)
+                                .foregroundColor(ColorManager.appPrimary)
+                                .padding(geometry.size.width * constants.paddingRatio)
+                                .shadow(radius: constants.shadowRadius)
+
+                            VStack {
+
+                                Image(systemName: Strings.LandingScreen.Images.plus)
+                                    .resizable()
+                                    .frame(width: geometry.size.width * constants.imageFrameRatio, height: geometry.size.width * constants.imageFrameRatio)
+                                    .foregroundColor(.white)
+                                Text(Strings.LandingScreen.plusButtonText)
+                                    .foregroundColor(.white)
+                            }
+                        }
+                    }
+
+                    NavigationLink(destination: MealListView(viewModel: .init(sdk: sdk))) {
+                        ZStack {
+                            RoundedRectangle(cornerRadius: constants.cornerRadius)
+                                .foregroundColor(ColorManager.appSecondary)
+                                .padding(geometry.size.width * constants.paddingRatio)
+                                .shadow(radius: constants.shadowRadius)
+                            VStack {
+                                Image(systemName: Strings.LandingScreen.Images.find)
+                                    .resizable()
+                                    .frame(width: geometry.size.width * constants.imageFrameRatio, height: geometry.size.width * constants.imageFrameRatio)
+                                    .foregroundColor(.white)
+                                Text(Strings.LandingScreen.findButtonText)
+                                    .foregroundColor(.white)
+                            }
+                        }
```

Comment:
Inside these NavigationLinks are the same other than a couple of colours, an image and text.
I'll have a go at refactoring these two